### PR TITLE
qoriq: remove usdpaa dtb and relocate sdk dtb

### PR DESCRIPTION
**Pull request by ting-liu**
To build with pure mainline kernel, update dtb in KERNEL_DEVICETREE
* usdpaa is an unsupported feature now, remove dtb for it.
* add QORIQ_DEFAULT_BSP definition to easily choose which BSP to use
* dtb with -sdk suffix only exist in kernel from NXP bsp, append them on NXP bsp only.

### CODE REVIEW

File: conf/machine/ls1043ardb.conf [hunk 1]
```diff
@@ -22,7 +22,10 @@ UBOOT_CONFIG ??= "tfa-secure-boot tfa"
 UBOOT_CONFIG[tfa] = "ls1043ardb_tfa_defconfig,,u-boot-dtb.bin"
 UBOOT_CONFIG[tfa-secure-boot] = "ls1043ardb_tfa_SECURE_BOOT_defconfig,,u-boot-dtb.bin"
 
-KERNEL_DEVICETREE ?= "freescale/fsl-ls1043a-rdb-sdk.dtb freescale/fsl-ls1043a-qds.dtb freescale/fsl-ls1043a-qds-sdk.dtb"
```
Comment (otavio) [inline conf/machine/ls1043ardb.conf]: Please don't. For NXP specific one, please use:

```
KERNEL_DEVICETREE_append_use-nxp-bsp = "  ... "
```

Comment (ting-liu) [reply conf/machine/ls1043ardb.conf]: so qoriq needs to set similar var as IMX_DEFAULT_BSP. can we rename IMX_DEFAULT_BSP to be more generic and move its definition from imx-base.inc? or define a new QORIQ_DEFAULT_BSP?

Comment (otavio) [reply conf/machine/ls1043ardb.conf]: Define a new one.

Comment (ting-liu) [reply conf/machine/ls1043ardb.conf]: define QORIQ_DEFAULT_BSP and use KERNEL_DEVICETREE_append_use-nxp-bsp. PR is updated.

**Pull request merged**